scripts/compile-resources.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+set -euxo pipefail
+
+CONTAINER_IMAGE="docker.io/andreatp/chicory-compilation-support"
```

Comment:
@bhelx if you wanna rebuild the image and publish it under the `dylibso` organization feel free to go ahead.

Reply:
Thanks I will take a look!